package.json

```diff
@@ -33,9 +33,10 @@
   "dependencies": {
     "@kiwicom/orbit-components": "^0.23.0",
     "@kiwicom/orbit-design-tokens": "^0.2.5",
+    "intl": "^1.2.5",
+    "react-native-shimmer": "^0.4.1",
```

Comment:
I don't know how I feel about adding new libraries, especially native ones.

Did you try with `Animated`?

Reply:
To get a `shimmer` effect we need `gradient`.  
* Creating our own gradient probably will produce a lot of `View`s which with animations can have performance implications. 
* [Ready gradient lid](https://github.com/react-native-community/react-native-linear-gradient) also has to be linked